project/plugins.sbt

```diff
@@ -15,7 +15,7 @@ addSbtPlugin("com.eed3si9n" % "sbt-unidoc" % "0.4.3")
 addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "3.0.0")
 addSbtPlugin("com.typesafe.sbt" % "sbt-site" % "1.4.0")
 // Akka gRPC -- sync with version in Dependencies.scala
-addSbtPlugin("com.lightbend.akka.grpc" % "sbt-akka-grpc" % "0.8.4")
+addSbtPlugin("com.lightbend.akka.grpc" % "sbt-akka-grpc" % "1.0.0")
```

Comment:
Or 1.0.1?

Reply:
1.0.1 doesn't (easily) allow adding external generators like play-grpc, so I figured it'd be a bit safer to stick with 1.0.0 for now.

Reply:
It would be great to update [the docs](https://doc.akka.io/docs/alpakka/2.0.1/google-cloud-pub-sub-grpc.html#build-setup), as well.